src/managers/UserAccessManager/useUserAccessManagerActions.js

```diff
@@ -91,10 +89,24 @@ export function useUserAccessManagerActions() {
 				enable: user.disabled ? '1' : '',
 			},
 		});
-
-		openLegacyModal({title: t('grid.user.disable')}, (closeData) => {
-			finishedCallback();
+		let currentRoles = '';
+		user.groups.forEach((group) => {
+			currentRoles = currentRoles + group.name + ',';
 		});
```

Comment:
This can be changed to:
```suggestion
		const currentRoles = user.groups.map(group => group.name).join(', ');
```
Note that space might be needed for a more structured message on the modal, when passing the description value to `openLegacyModal` function call below.